docs/reddit-migration/strategy/reads-and-writes.md

```diff
@@ -0,0 +1,36 @@
+# Migrate Reads and Writes
+
+We need to gracefully migrate our APIs over to the new schema. We want to avoid or at the very least minimize any sort of downtime.
+
+In general, migrating reads and writes can happen in a siloed manner, in order of highest to lowest level in the relationship hierarchies:
+
+- Channels
+- Posts
+- Comments
+- etc...
+
+### Add writes to the new tables
+
+This should be fairly straightforward, we should add another write to where we are currently writing (calling APIs) to reddit.
```

Comment:
Curious if this will be part of a transaction, with the call to reddit?

Reply:
Where we have explicit transactions already these should be added as part of those yes, there are some places where that hasn't been possible or valuable so it's ok in those places not to have a transaction. I believe they're documented as such in the code.

Reply:
I think it would be worth saying that explicitly here, because I can see this question coming up again.